app/translations/eo/LC_MESSAGES/messages.po

```diff
@@ -754,131 +754,131 @@ msgstr "Here we'r giein tha bak an forrits o why we maun hae yer wittins. Forbye
-msgstr ""
+msgstr "Throch tha preein we'll tak oan ootbye consarns tae gie iz a heeze wi pairts o tha traffeckin. Thir ootbye consarns daes thair darg unnèr oor airtin."
 
 #: templates/static/privacy.html:30
 msgid "We treat the information we hold with respect, keeping it secure and confidential. Data may be made available to approved researchers where there is a clear public value and it’s safe and lawful to do so. More information on <a href='{policies_url}'>how we allow access to data for research</a> is available. We also publish a full list of all the <a href='{researchers_url}'>approved researchers</a> who we’ve granted access to."
-msgstr ""
+msgstr "We guide tha wittins we haud wi mense, haudin it siccar an hiddlie. Wittins cud be pit tae tha han o homologatit bodies o skeel whar thar's a straucht-forrit guid fur resydentèrs an it's siccar an no agin tha laa tae dae't. Mair wittins anent <a href='{policies_url}'>whut wye we lat ingang til wittins fur screengin</a> is pit tae yer han. Forbye, we ootset a fu leet o aa tha <a href='{researchers_url}'>homologatit bodies o skeel</a> 'at we hae lat ingang it."
 
 #: templates/static/privacy.html:33
 msgid "How long are personal data kept?"
-msgstr ""
+msgstr "Hoo lang wud wittins anent a bodie's sel be hauden?"
 
 #: templates/static/privacy.html:35
 msgid "Data protection law requires that personal data are kept for no longer than is needed to fulfil the purposes for which they were originally collected."
-msgstr ""
+msgstr "Tha laa anent tha fennin o wittins gars fowk haud wittins anent a bodie's sel fur nae langer nor whut's necessar tae shuit tha ettlins the' wur ingaithert fur at tha furst."
 
 #: templates/static/privacy.html:37
 msgid "The law allows that information held for statistical purposes only, such as information from your questionnaire, may be kept for longer periods. We’ll only continue to hold personal data where they are still used to produce statistics. Data used by approved researchers will be held within our <a href='{url}'>Secure Research Service</a>. Any personal data that we do not need, such as names and addresses, will be removed from the data at the earliest opportunity. The data will undergo regular reviews and will be deleted when they are no longer required."
-msgstr ""
+msgstr "Tha laa gies lave 'at wittins hauden fur nummerin ettlins alane, tha like o wittins frae yer spierin foarm, micht be hauden fur langer tacks. We'll jist haud tae hingin oan tae wittins anent a bodie's sel whar the'r bein uised yit tae rise nummerin. Wittins uised wi homologatit bodies o skeel wull be hauden inbye oor <a href='{url}>Siccar Screengin Fennin</a>. Onie wittins anent a bodie's sel we dinnae be tae hae, tha like o whut the' cry fowk an thair bakkins, wull be taen oot o tha pose o wittins at tha furst inlat. We'r fur haein reglar scances o tha wittins pose an blorryin it oot whan we dinnae be tae hae it onie mair."
```

Comment:
The href in this text is missing a closing quote, when rendered:
![image](https://user-images.githubusercontent.com/1754822/64977130-b8eb9e80-d8aa-11e9-9e98-c97315f74812.png)